src/Card/CardStyles.ts

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+import colors from 'src/styles/lakefrontColors';
+
+export const CardContentContainer = styled.div(() => ({
+    display: 'inline-grid',
+    gridTemplateColumns: '1fr auto',
+    gridTemplateRows: '55px auto auto',
+    width: 250,
+    height: 'auto',
+    border: `1px solid ${colors.alto}`,
+    borderRadius: 3,
+    padding: 10,
+    margin: 10,
```

Comment:
Let's take the margin off this. It should be up to the parent/implementation to determine if spacing should be around it or now.